public_data/management/commands/config_load_ocsge.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Docstring qui précise complètement comment renseigner le fichier

- KISS => keep it simple stupid => on doit comprendre en lisant (philosophie Python, on écrit 1 fois on lit 4 fois)
- un seul format json par typologie d'import, éviter la simplification du fichier config qui alourdi le code tout en le rendant plus difficile à lire pour un nouvel arrivant
- pas d'information importante dans les clés, ça rend difficile la compréhension du fichier, ça crée

```json
{ 
    "Essone": {
        "code_departement": 77,
        "layers":[
            {
                "type": "ocsge",
                "path": "essonne_ocsge_2018.zip",
                "year": 2018,
                "mapping": { 
                    "mpoly": "MULTIPOLYGON"
                }
            },
            {
                "type": "built_area",
                "path": "essonne_zone_construite_2018.zip",
                "year": 2018
            },
            {
                "type": "diff",
                "path": "essonne_diff_2018_2021.zip",
                "year_old": 2018,
                "year_new": 2021
            },
            { ... }
        ]
   }
}
```